docker/volumes/db/init/01-auth-schema.sql

```diff
@@ -94,32 +94,38 @@ returns uuid
 language sql stable
 as $$
   select 
+  nullif(
   	coalesce(
 		current_setting('request.jwt.claim.sub', true),
 		(current_setting('request.jwt.claims', true)::jsonb ->> 'sub')
-	)::uuid
+	)
```

Comment:
This should be `nullif(coalesce(...), '')::uuid`—`nullif` accepts two arguments, and returns `null` if they match: https://www.postgresql.org/docs/14/functions-conditional.html#FUNCTIONS-NULLIF.